### PR TITLE
Add Wifi packages and systemd services

### PR DESCRIPTION
**Pull request by matinlotfali**
For some reason, the other pull request was showing older commits in it. So I closed the other one and created this one.

This PR needs more commits to add the systemd services.

### CODE REVIEW

**Review by mistyron — COMMENTED** [review]
requested clarification for change in misty-image.bb

File: recipes-core/images/mistysom-image.bb [hunk 1]
```diff
@@ -7,7 +7,7 @@ LICENSE = "MIT"
 inherit core-image features_check
 inherit extrausers
 
-DISTRO_FEATURES = "x11 opengl"
+REQUIRED_DISTRO_FEATURES = "x11 opengl"
```
[the remark this answers is not in the document]
Comment (matinlotfali) [reply recipes-core/images/mistysom-image.bb:10]: No, previously you were unhappy with the change from REQUIRED_DISTRO_FEATURES to DISTRO_FEATURES. So I reverted the change.

[the remark this answers is not in the document]
Comment (mistyron) [reply recipes-core/images/mistysom-image.bb:10]: That is correct, I'm just surprised to see it in this PR, it shoudl hvae been in the same where `DISTRO_FEATURES1 got added to local.conf but this is okay! :) Thanks!

**Review by mistyron — APPROVED** [review]
LGTM

**Review by mistyron — APPROVED** [review]
LGTM

**Pull request merged**